Include section on post processing of gene expression data in processing information part of docs (#97)
AlexsLemonade/scpca-docs: docs/processing_information.md

Briefly, selective alignment uses a mapping score validated approach to identify maximal exact matches between reads and the provided index.
For all samples, we used selective alignment to the `splici` index.

> A more detailed description of the mapping strategy invoked by `salmon` in conjunction with `alevin-fry` can be found in [Srivastava _et al._ 2020](https://doi.org/10.1186/s13059-020-02151-8).

Reviewer: My internal style guide (based on Genetics) says to leave the date in parens here.

Author: The reason I changed this is because when I was adding the new reference I went to see what our preferred style was here and this was the only reference in this doc where we were using parenthesis. All the other references don't have them. Do we want to change all of them to have parenthesis then?

Reviewer: The difference here is that `Srivastava _et al._` is part of the sentence that the reference appears in, so I only put the year in parenthesis. In the other cases, the entire reference is in parenthesis (except for the reference you added, where the names are again part of the sentence).

For some libraries, `DropletUtils::emptyDropsCellRanger()` may fail due to low numbers of droplets with reads or other violations of its assumptions.
For these libraries, only droplets containing at least 100 UMI are included in the filtered counts matrix.

### Processed gene expression data

> In addition to the raw gene expression data, we also provide a `_processed.rds` file containing a `SingleCellExperiment` object with the filtered, normalized counts matrix and embeddings from dimensionality reduction.

Reviewer: Some suggested edits here. (I don't love "embeddings" as I think it might be confusing, so I went with something much more generic).
```suggestion
In addition to the raw gene expression data, we also provide a `_processed.rds` file containing a `SingleCellExperiment` object with further filtering applied, a normalized counts matrix, and results from dimensionality reduction.
```

Prior to normalization, low-quality cells are removed from the gene by cell counts matrix.

> To predict low-quality cells, we use [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html), a package that jointly models proportion of reads belonging to mitochondrial genes and number of unique genes detected.

Reviewer: ```suggestion
To identify low-quality cells, we use [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html), a package that jointly models proportion of reads belonging to mitochondrial genes and number of unique genes detected.
```

> Cells with a high likelihood of being compromised, greater than 0.75, and cells that do not pass a minimum number of unique genes detected threshold of 200 are removed from the counts matrix present in the `_processed.rds` file.

Reviewer: I like parenthesis here, but either way.
```suggestion
Cells with a high likelihood of being compromised (greater than 0.75) and cells that do not pass a minimum number of unique genes detected threshold of 200 are removed from the counts matrix present in the `_processed.rds` file.
```

> Log-normalized counts are calculated using the deconvolution method presented in [Lun, Bach, and Marioni 2016](https://doi.org/10.1186/s13059-016-0947-7).

Reviewer: Style!
```suggestion
Log-normalized counts are calculated using the deconvolution method presented in [Lun, Bach, and Marioni (2016)](https://doi.org/10.1186/s13059-016-0947-7).
```

> The log-normalized counts are used to model variance of each gene prior to selecting the top 2000 highly variable genes(HVG).

Reviewer: ```suggestion
The log-normalized counts are used to model variance of each gene prior to selecting the top 2000 highly variable genes (HVGs).
```

> The HVGs are then used as input to principal component analysis, and the top 50 principal components are selected.
> Finally, the principal components are used to calculate the [UMAP (Uniform Manifold Approximation and Projection)](http://bioconductor.org/books/3.13/OSCA.basic/dimensionality-reduction.html#uniform-manifold-approximation-and-projection) embeddings.

Reviewer: ```suggestion
These HVGs are then used as input to principal component analysis, and the top 50 principal components are selected.
Finally, these principal components are used to calculate the [UMAP (Uniform Manifold Approximation and Projection)](http://bioconductor.org/books/3.13/OSCA.basic/dimensionality-reduction.html#uniform-manifold-approximation-and-projection) embeddings.
```
